src/templates/devices.html

```diff
@@ -1,12 +1,14 @@
 {% extends "base.html" %}
 {% load static %}
+{% load crispy_forms_tags %}
 
 {% block content %}
 <div class="container">
 	<h1>Devices</h1>
-	<form action="{% url 'devices' %}" method="get">
-		<input name="q" type="text" placeholder="Search device...">
-	</form>
+	 <form action="{% url 'devices' %}" method="get">
+        {{ search_form | crispy }}
```

Comment:
it would be nice, if the the form was implemented using nice looking bootstrap dropdown menu 
https://getbootstrap.com/docs/4.0/components/dropdowns/